### ISSUE TITLE
Docker: Error while executing 'docker run hello-world' command

### ISSUE THREAD

**Issue by shwetaaher** [moby/moby#30360]
I have installed docker toolbox for Windows 7 64 bit system. I am getting following error while executing 'docker run hello-world' command .
Please guide me to resolve this.
Thanks in advance!

```
$ docker run hello-world  
Unable to find image 'hello-world:latest' locally
latest: Pulling from library/hello-world
78445dd45222: Pulling fs layer
C:\Program Files\Docker Toolbox\docker.exe: error pulling image configuration: G
et https://dseasb33srnrn.cloudfront.net/registry-v2/docker/registry/v2/blobs/sha
256/48/48b5124b2768d2b917edcb640435044a97967015485e812545546cbed5cf0233/data?Exp
ires=1484910885&Signature=DsT7p2RzmUsymlHz2qG51UJspf9ZvTvHF2pjxJPEi1~c9SomNSk2dv
Sqn1UEpsuse3iVmkq37HKT7cEvRaPsu5vB1RlHBgUg6FJB0DGukLBIq1t2hddVPvlI4Q7lm5B5j3NzRI
vj-hYXrAYDxKAM3TX6LsIdrSrmTHlyIZJ4~1A_&Key-Pair-Id=APKAJECH5M7VWIS5YZ6Q: read tc
p 10.0.2.15:40664->52.222.190.31:443: read: connection reset by peer.
See 'C:\Program Files\Docker Toolbox\docker.exe run --help'.
```

**Comment by thaJeztah** [comment]
This looks like a networking issue in your setup. Could you at least provide the information that was requested in the issue template (https://raw.githubusercontent.com/docker/docker/master/.github/ISSUE_TEMPLATE.md)?

Also;

- Does your network use a proxy server?
- What version of VirtualBox do you have installed? There has been a bug in certain versions of VirtualBox that resulted in https connections to be broken

**Comment by shwetaaher** [comment]
**Description**
I have installed Docker Toolbox on Windows 7-64 bit system

**Steps to reproduce the issue:**
1. Download and install the Docker Toolbox.
2. After successful installation, double click on 'Docker Quickstart terminal'.
3. After configuring 'default' docker machine, I have executed the 'docker run hello-world' command.
4. On executing this command I got the following error.
{{
$ docker run hello-world  
Unable to find image 'hello-world:latest' locally
latest: Pulling from library/hello-world
78445dd45222: Pulling fs layer
C:\Program Files\Docker Toolbox\docker.exe: error pulling image configuration: G
et https://dseasb33srnrn.cloudfront.net/registry-v2/docker/registry/v2/blobs/sha
256/48/48b5124b2768d2b917edcb640435044a97967015485e812545546cbed5cf0233/data?Exp
ires=1484910885&Signature=DsT7p2RzmUsymlHz2qG51UJspf9ZvTvHF2pjxJPEi1~c9SomNSk2dv
Sqn1UEpsuse3iVmkq37HKT7cEvRaPsu5vB1RlHBgUg6FJB0DGukLBIq1t2hddVPvlI4Q7lm5B5j3NzRI
vj-hYXrAYDxKAM3TX6LsIdrSrmTHlyIZJ4~1A_&Key-Pair-Id=APKAJECH5M7VWIS5YZ6Q: read tc
p 10.0.2.15:40664->52.222.190.31:443: read: connection reset by peer.
See 'C:\Program Files\Docker Toolbox\docker.exe run --help'.
}}


>> Does your network use a proxy server?
No.

>>What version of VirtualBox do you have installed?
Version of VirtualBox: 5.1.10
version of Docker Toolbox: 1.12.6

**Comment by thaJeztah** [comment]
It does look very similar to https://github.com/docker/docker/issues/27734, which should've been fixed in VirtualBox 5.1.10, but I see the issue has been reopened in the VirtualBox issue tracker  https://www.virtualbox.org/ticket/16084

**Comment by sowmyapappu** [comment]
I installed docker on centos 7 machine. I am unable to run the hello-world application.

[root@localhost ~]# docker run hello-world
Unable to find image 'hello-world:latest' locally
Trying to pull repository docker.io/library/hello-world ...
latest: Pulling from docker.io/library/hello-world
78445dd45222: Pulling fs layer
/usr/bin/docker-current: error pulling image configuration: Get https://dseasb33cb640435044a97967015485e812545546cbed5cf0233/data?Expires=1488526869&Signature=AWmk0JKGJRFcilFHIpgpxspqxnOcIENONYPq~ZFI0MRqJRuBCohA1vNXESRVEs8VwTirrUW~OEJl0aV3J




]# docker info
Containers: 0
 Running: 0
 Paused: 0
 Stopped: 0
Images: 0
Server Version: 1.12.5
Storage Driver: devicemapper
 Pool Name: docker-253:0-134531172-pool
 Pool Blocksize: 65.54 kB
 Base Device Size: 10.74 GB
 Backing Filesystem: xfs
 Data file: /dev/loop0
 Metadata file: /dev/loop1
 Data Space Used: 11.8 MB
 Data Space Total: 107.4 GB
 Data Space Available: 49.38 GB
 Metadata Space Used: 581.6 kB
 Metadata Space Total: 2.147 GB
 Metadata Space Available: 2.147 GB
 Thin Pool Minimum Free Space: 10.74 GB
 Udev Sync Supported: true
 Deferred Removal Enabled: false
 Deferred Deletion Enabled: false
 Deferred Deleted Device Count: 0



 docker version
Client:
 Version:         1.12.5
 API version:     1.24
 Package version: docker-common-1.12.5-14.el7.centos.x86_64
 Go version:      go1.7.4
 Git commit:      047e51b/1.12.5
 Built:           Mon Jan 23 15:35:13 2017
 OS/Arch:         linux/amd64

Server:
 Version:         1.12.5
 API version:     1.24
 Package version: docker-common-1.12.5-14.el7.centos.x86_64
 Go version:      go1.7.4
 Git commit:      047e51b/1.12.5
 Built:           Mon Jan 23 15:35:13 2017
 OS/Arch:         linux/amd64

**Comment by thaJeztah** [comment]
Closing this because of age
